common/src/main/java/tachyon/conf/TachyonConf.java

```diff
@@ -219,6 +219,8 @@ public void merge(TachyonConf alternateConf) {
 
   // TODO(binfan): this method should be hidden and only used during initialization and tests.
   public void set(String key, String value) {
+    Preconditions.checkArgument(key != null && value != null,
+        String.format("the key value pair (%s, %s) cannot have null value", key, value));
```

Comment:
The message is not quite correct, since the precondition states the key cannot be null right? Or maybe I'm misinterpreting the meaning of the second `value`?